Sources/Danger/BitBucketCloud.swift

```diff
@@ -33,7 +33,7 @@ extension BitBucketCloud {
             case createdOn = "created_on"
             case description
             case destination
-            case id // swiftlint:disable:this identifier_name
+            case id 
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/BitBucketCloud.swift;
  Line: 36;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)